test/check-machines-lifecycle

```diff
@@ -161,13 +161,13 @@ class TestMachinesLifecycle(machineslib.VirtualMachinesCase):
 
         # reboot
         self.machine.execute(f"echo '' > {args['logfile']}")
```

Comment:
Let's please move that into `performAction`. It's easy to forget, and apparently it's an assumption.